client/src/services/userService.js

```diff
@@ -9,5 +9,14 @@ export default {
             }
         });
         return res.data || [];
+    },
+    submitUser: async (email, passwort) => {
+        let res = await axios.post('/api/user', {
```

Comment:
before here was implemented a call with GET... it's actually wrong cause submitting of Form happens with POST Request))